demos/total_radiation.py

```diff
@@ -1,28 +1,49 @@
 
+import numpy as np
 import matplotlib.pyplot as plt
-plt.ion()
 
 from cherab.core.atomic import neon
+from cherab.core.math import sample2d_grid
 from cherab.adas import ADAS
 
 
-print("testing total radiation for Neon")
+print("Testing total radiation for Neon.")
 
 atomic_data = ADAS()
 
-plt.figure()
+electron_density = np.array([1.e19])
+electron_temperature = np.logspace(0, 4, 41)
 
-ne_total_rad = atomic_data.radiated_power_rate(neon, 'total')
-ne_total_rad.plot_temperature()
+# Obtaining total radiation using ADAS405 code (equilibrium balance)
+total_rad = atomic_data.total_radiated_power(neon)
+total_power = sample2d_grid(total_rad, electron_density, electron_temperature).squeeze()
 
-ne_line_rad = atomic_data.radiated_power_rate(neon, 'line')
-ne_line_rad.plot_temperature()
+# Obtaining fractional abundance using ADAS405 code
+fractional_abundance = np.zeros((11, electron_temperature.size))
+for charge in range(11):
+    fraction = atomic_data.fractional_abundance(neon, charge)
+    fractional_abundance[charge] = sample2d_grid(fraction, electron_density, electron_temperature).squeeze()
 
-ne_continuum_rad = atomic_data.radiated_power_rate(neon, 'continuum')
-ne_continuum_rad.plot_temperature()
+# Obtaining line and continuum radiation
+line_power = np.zeros((10, electron_temperature.size))
+continuum_power = np.zeros((10, electron_temperature.size))
+for charge in range(10):
```

Comment:
Same comment about avoiding hard-coding the value 10.

Reply:
Done.